apps/io-profile-async/src/utils/appinsights.ts

```diff
@@ -0,0 +1,57 @@
+import * as ai from "applicationinsights";
+import {
+  EventTelemetry,
+  ExceptionTelemetry
+} from "applicationinsights/out/Declarations/Contracts";
+import * as E from "fp-ts/lib/Either";
+import * as O from "fp-ts/lib/Option";
+import { initAppInsights } from "@pagopa/ts-commons/lib/appinsights";
+import { IntegerFromString } from "@pagopa/ts-commons/lib/numbers";
+import { NonEmptyString } from "@pagopa/ts-commons/lib/strings";
+import { pipe } from "fp-ts/lib/function";
+
+// the internal function runtime has MaxTelemetryItem per second set to 20 by default
+// @see https://github.com/Azure/azure-functions-host/blob/master/src/WebJobs.Script/Config/ApplicationInsightsLoggerOptionsSetup.cs#L29
+const DEFAULT_SAMPLING_PERCENTAGE = 20;
```

Comment:
I would suggest to set this to 5 to align it to other microservices.
```suggestion
const DEFAULT_SAMPLING_PERCENTAGE = 5;
```